packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -878,6 +890,16 @@ class CupertinoTextField extends StatefulWidget {
     properties.add(DiagnosticsProperty<Clip>('clipBehavior', clipBehavior, defaultValue: Clip.hardEdge));
     properties.add(DiagnosticsProperty<bool>('scribbleEnabled', scribbleEnabled, defaultValue: true));
     properties.add(DiagnosticsProperty<bool>('enableIMEPersonalizedLearning', enableIMEPersonalizedLearning, defaultValue: true));
+    properties.add(DiagnosticsProperty<List<String>>('contentCommitMimeTypes', contentCommitMimeTypes,
+        defaultValue: onContentCommitted == null ? null : <String>[
+          'image/png',
+          'image/bmp',
+          'image/jpg',
+          'image/tiff',
+          'image/gif',
+          'image/jpeg',
+          'image/webp'
+        ]));
```

Comment:
Double-check me please, but I think since both the Cupertino and Material TextFields ultimately just implement EditableText, it's not required. EditableText sets the defaults here: https://github.com/BlueBubblesApp/flutter/blob/3cb80909b6bb92585f202ab73165237fbe7ea038/packages/flutter/lib/src/widgets/editable_text.dart#L602-L611

Also, I remember trying to set the default MIMETypes specifically for the Cupertino and Material Text Fields, but the Dart analyzer was throwing errors before